backend/app/Services/WorkOrder/WorkOrderStopService.php

```
@@ -0,0 +1,293 @@
+     *
+     * @param  array<string, mixed>  $data
+     */
+    protected function openDowntime(WorkOrder $workOrder, array $data, User $user): ?ProductionDowntime
+    {
+        $reasonId = $data['downtime_reason_id'] ?? null;
+
+        if ($reasonId === null || $workOrder->line_id === null) {
+            return null;
+        }
+
+        return ProductionDowntime::create([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\ProductionDowntime::create().


(staticMethod.notFound)